src/workers/notifications/postAdded.ts

```diff
@@ -49,6 +49,13 @@ const worker: NotificationWorker = {
       return;
     }
 
+    if (
+      source.type === SourceType.Machine &&
+      (post.flags as PostFlags)?.showOnFeed === false
```

Comment:
do check if they are `string`, because from cdc they can be as type suggests

Reply:
Parsing as string now.